src/envs/pybullet_cover.py

```diff
@@ -0,0 +1,313 @@
+"""A PyBullet version of Cover."""
+
+from typing import ClassVar, Dict, List, Sequence, Tuple
+
+import numpy as np
+import pybullet as p
+from gym.spaces import Box
+
+from predicators.src import utils
+from predicators.src.envs.cover import CoverEnv
+from predicators.src.envs.pybullet_env import PyBulletEnv, \
+    create_pybullet_block
+from predicators.src.envs.pybullet_robots import _SingleArmPyBulletRobot, \
+    create_single_arm_pybullet_robot
+from predicators.src.settings import CFG
+from predicators.src.structs import Action, Array, Object, \
+    ParameterizedOption, Pose3D, State, Type
+
+
+class PyBulletCoverEnv(PyBulletEnv, CoverEnv):
+    """PyBullet Cover domain."""
+    # Parameters that aren't important enough to need to clog up settings.py
+
+    # Option parameters.
+    _open_fingers: ClassVar[float] = 0.04
```

Comment:
we don't need to do this now, but i'm guessing we will wanna do some refactoring that moves this into the robots, since these refer to joint values that may be different for other robots